pkg/scalers/postgresql_scaler.go

```diff
@@ -4,14 +4,15 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	_ "github.com/lib/pq"
+	"strconv"
+
+	"github.com/lib/pq"
```

Comment:
```suggestion

```
According to go vet this one is imported but unused. @dimberman do you know if this can be removed?

Reply:
Done

Reply:
Just wanted to confirm if its only my machine or CI fails as well

Reply:
@kashifsoofi I think this import should stay as it was `    _ "github.com/lib/pq"` as this is a required PostgreSQL drive.